crates/control/src/role_assignment.rs

```diff
@@ -35,10 +39,9 @@ use crate::localization::generate_initial_pose;
 pub struct RoleAssignment {
     last_received_spl_striker_message: Option<SystemTime>,
```

Comment:
This can never be `None`, can it? This is only `None` in construction and the first cycle immediately sets a `Some` to it because `self.role_initialized == false`